meteor/client/lib/rundownTiming.ts

```diff
@@ -903,3 +909,68 @@ function getSegmentRundownAnchorFromPart(
 
 	return nextRundownAnchor
 }
+
+export type MinimalPartInstance = Pick<
+	PartInstance,
+	| '_id'
+	| 'isTemporary'
+	| 'rundownId'
+	| 'segmentId'
+	| 'segmentPlayoutId'
+	| 'takeCount'
+	| 'part'
+	| 'timings'
+	| 'orphaned'
+>
+
+export function findPartInstancesInQuickLoop(
+	playlist: DBRundownPlaylist,
+	sortedPartInstances: MinimalPartInstance[]
+): Record<TimingId, boolean> {
+	const partsInQuickLoop: Record<TimingId, boolean> = {}
+	if (
+		!isLoopDefined(playlist) ||
+		isLoopLocked(playlist) // a crude way of disabling the dots when looping the entire playlist
+	) {
+		return partsInQuickLoop
+	}
```

Comment:
Shouldn't this use some sort of a `isLoopOnEntirePlaylist`? Also, wouldn't it be more resilient to just say `return {}`, in case someone puts some logic before this `if` block?